### DISCUSSION TITLE
Trouble getting TailPress to work

### DISCUSSION THREAD

**Discussion by muhammadfuaad** [tailpress/tailpress#205] [Q&A]
I've added Tailpress in my WordPress website to use Tailwind, but it's not working. Certainly, I'm making a mistake and that's why I'm asking how to check for the mistake. 
My browser is considering Tailwind classes as bootstrap classes and applying the styles from bootstrap css source file(as they look similar). How could I tell my browser that these are tailwind classes not bootstrap.
Do we only need to install and activate tailpress and start adding our code in the theme file editor? Or there is some other configuration needed as well?
And strangely, tailwind classes are still working for "Twenty Twenty-three" theme even without Tailpress.

**Comment by jeffreyvr** [comment]
Are you sure you are using this TailPress? I ask because unfortunately, there is a plugin that picked the same name.

If you are using this theme boilerplate, can you tell me a bit more on how you got started?
